src/components/App.css

```diff
@@ -1,19 +1,77 @@
 .App {
-    display: grid;
-    grid-template-rows: 1fr auto 1fr;
-    height: 100vh;
+  display: grid;
+  grid-template-rows: 1fr auto 1fr;
+  grid-template-columns: 1fr;
+  height: 100vh;
+}
+
+.App--four {
+  grid-template-columns: 1fr 1fr;
 }
 
 .controls {
-    align-self: center;
-    justify-self: center;
-    display: grid;
-    justify-items: center;
-    grid-auto-flow: column;
+  align-self: center;
+  justify-self: center;
+  display: grid;
+  justify-items: center;
+  grid-column: 1 / span 2;
+  grid-auto-flow: column;
+}
+
+.App--four button div {
+  font-size: 70px;
+}
+
+@media (min-width: 350px) {
```

Comment:
this will look much simpler when we switch to SASS  or post-css plugin since we will be able to nest classes but let's leave it as it is for now